### DISCUSSION TITLE
Range extender DHCP

### DISCUSSION THREAD

**Discussion by toastbrot69** [arendst/Tasmota#16047] [Ideas]
Hi, 
its about tasmota 12.0.2:
Range-extender in routings mode is working fine, thanks a lot!

Feature request:
If using dhcp on the remote/extender side of tasmota, there seems to be no way to find out the assigned addresses.
So if i add other tasmotas (without serial port) to the extender-network (routing-mode), I have to manually scan the extender-net for the assigned addresses.
Something like “RgxDhcp” for viewing all subscriptions…
Regards

**Comment by barbudor** [comment]
Every tasmota publishes 3 informative messages at boot. One of those include the ipaddress

**Reply by sfromis** [reply]
While the range extender node does not have a function to list the DHCP assignments, the IP address is additionally available in the discovery payload published at boot, and in the output of commands `Status 5` and `IPAddress`. You could assign a group topic to the nodes you put on a range extender, and then you can publish one single command to have all of them report their IP address.

**Comment by toastbrot69** [comment]
Thanks for your replies, but all these infos are available _once_ you found out the address of a node and then configurated mqtt.
But before configuration, or with mqtt-less nodes that’s not working.

**Reply by sfromis** [reply]
You could make the MQTT config, including group topic, before connecting to the range extender network. Your first network setup could be regular wifi, and then reconfig for rgx after setup. As you cannot config for the rgx network "before configuration", there is always a place where you have the access.

Anyway, the project is open source, and contributions are generally invited :smile:

**Reply by toastbrot69** [reply]
I used the build in hotspot for configuration(so no pre-configuration), after setting the extender-WiFi params there’s an immediate restart (no possibility to config mqtt), if you are not fast enough remembering the ip you are lost

**Reply by sfromis** [reply]
As I wrote, you can connect to your regular wifi first, do the config, and after that connect to the rgx network.

**Reply by barbudor** [reply]
Or just be lazy and build your own binary with wifi credentials and mqtt settings hard coded in user_config_override.h

**Comment by joba-1** [comment]
still need this? see https://github.com/arendst/Tasmota/pull/17048, comments welcome :)
